explorer/rest/tests/test_rest.py

```diff
@@ -59,15 +59,21 @@ def client(app):  # pylint: disable=redefined-outer-name
 # endregion
 
 
+def _assert_status_code_and_headers(response, expected_status_code):
+	# Assert:
```

Comment:
can remove this since name of function is clear that this is part of assert stage